airflow/providers/amazon/aws/sensors/cloud_formation.py

```diff
@@ -18,6 +18,11 @@
 """This module contains sensors for AWS CloudFormation."""
 from typing import Optional
 
+try:
+    from functools import cached_property
+except ImportError:
+    from cached_property import cached_property
```

Comment:
Is there a reason why we need both imports?

Reply:
If this is an issue then we probably should fix all other operators that use `from cached_propert`, WDYT?

Reply:
We do, see https://github.com/apache/airflow/pull/14606

Reply:
Yup, like Elad mentioned  #14606 removed the dep for Python 3.8 users. this just build on top of that